float64 yaw_rad
float64 yaw_rate
float64 yaw_x_position
float64 yaw_y_position